packages/adapter-static/index.js

```diff
@@ -11,28 +11,28 @@ export default function ({ pages = 'build', assets = pages, fallback, precompres
 	return {
 		name: '@sveltejs/adapter-static',
 
-		async adapt(builder) {
-			builder.rimraf(assets);
-			builder.rimraf(pages);
+		async adapt({ utils }) {
+			utils.rimraf(assets);
```

Comment:
I think when you destructure object like that, you dont get the `utils`, you get whats inside of them.
Take a look here: https://runkit.com/pavelloz/utils